pom.xml

```diff
@@ -76,8 +80,30 @@
             <artifactId>jansi</artifactId>
             <version>${version.org.fusesource.jansi}</version>
         </dependency>
+
+        <dependency>
+            <groupId>org.projectlombok</groupId>
+            <artifactId>lombok</artifactId>
+            <version>${version.org.projectlombok.lombok}</version>
+            <scope>provided</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.slf4j</groupId>
+            <artifactId>slf4j-simple</artifactId>
+            <version>${version.org.slf4j.slf4j-simple}</version>
+        </dependency>
     </dependencies>
 
+    <dependencyManagement>
```

Comment:
yup, we will need to put all deps to dependencyManagement in root pom and define dependencies in each module